src/ctapipe/instrument/optics.py

```diff
@@ -263,3 +271,172 @@ def __repr__(self):
 
     def __str__(self):
         return self.name
+
+
+class PSFModel(TelescopeComponent):
+    """
+    Base component to describe image distortion due to the optics of the different cameras.
+    """
+
+    @u.quantity_input(x=u.m, y=u.m, x0=u.m, y0=u.m)
+    @abstractmethod
+    def pdf(self, x, y, x0, y0) -> np.ndarray:
+        """
+        Calculates the value of the psf at a given location
+
+        Parameters
+        ----------
+        x : u.Quantity[length]
+            x-coordinate of the point on the focal plane where the psf is evaluated
+        y : u.Quantity[length]
+            y-coordinate of the point on the focal plane where the psf is evaluated
+        x0 : u.Quantity[length]
+            x-coordinate of the point source on the focal plane
+        y0 : u.Quantity[length]
+            y-coordinate of the point source on the focal plane
+        Returns
+        ----------
+        psf : np.ndarray
+            value of the PSF at the specified location with the specified position of the point source
+        """
+
+        pass
+
+
+def _cartesian_to_polar(x, y):
+    r = np.sqrt(x**2 + y**2)
+    phi = np.arctan2(y, x)
+    return r, phi
+
+
+class ComaPSFModel(PSFModel):
+    r"""
+    PSF model describing pure coma aberrations PSF effect.
+
+    The PSF is described by a combination of an asymmetric Laplacian for the radial part and a symmetric Laplacian in the polar direction.
+    It uses an asymmetric Laplacian for the radial part
+
+    .. math:: f_{R}(r, K) = \begin{cases}\frac{1}{S_{R}(K+K^{-1})}e^{-K\frac{r-L}{S_{R}}}, r\ge L\\ \frac{1}{S_{R}(K+K^{-1})}e^{\frac{r-L}{KS_{R}}}, r < L\end{cases}
```

Comment:
Probably better to rename L, the breakpoint of the asymmetric Laplacian, to r_0 in line with the terminology of the code?